compiler/modules/print.llvm/common.bal

```diff
@@ -103,3 +103,36 @@ public type GlobalProperties record {|
     Linkage linkage = "external";
 |};
 
+// Corresponds to LLVMDWARFSourceLanguage
+public type DWARFSourceLanguage "C99";
+
+// Corresponds to LLVMDWARFEmissionKind
+public type EmissionKind "none"|"full"|"lineTablesOnly";
+//TODO: add the other options
+public type CompileUnitProperties record {|
+    DWARFSourceLanguage language = "C99";
+    Metadata file;
+    boolean isOptimized = false;
+    int runtimeVersion = 0;
+    EmissionKind kind = "full";
+|};
+
+public type ModuleFlag ["Debug Info Version", int]|["Dwarf Version", int];
+// Corresponds to LLVMModuleFlagBehavior
+public type ModuleFlagBehavior "error"|"warning"|"require"|"override"|"append"|"appendUnique"|"max";
+
+// Corresponds to LLVMDIFlags
+public type DIFlag "zero";
+public type FunctionMetadataProperties record {|
+    Metadata? scope;
+    string? name;
+    string? linkageName;
+    Metadata? file;
+    int lineNo;
+    Metadata? ty;
+    "localToUnit"? localToUnit = "localToUnit";
```

Comment:
Just use booleans for field values (not the same situation as parameters).

Reply:
Fixed